src/js/containers/agency/visualizations/ObligationsByAwardTypeContainer.jsx

```diff
@@ -35,7 +35,7 @@ export default function ObligationsByAwardTypeContainer({ fiscalYear, windowWidt
             obligationsByAwardTypeRequest.current.cancel();
         }
         dispatch(resetAwardObligations());
-    }, []);
+    }, [dispatch]);
```

Comment:
do you want this one there too?

Reply:
That one doesn't seem to be affecting anything. The others were breaking things, so I added the ignore line.